scene/gui/color_picker.cpp

```diff
@@ -2234,8 +2236,11 @@ ColorPicker::ColorPicker() {
 	add_mode(new ColorModeRAW(this));
 	add_mode(new ColorModeOKHSL(this));
 
+	sliders_vbox = memnew(VBoxContainer);
+	real_vbox->add_child(sliders_vbox);
+
 	mode_hbc = memnew(HBoxContainer);
-	real_vbox->add_child(mode_hbc);
+	sliders_vbox->add_child(mode_hbc);
```

Comment:
This changes the behavior of `sliders_visible` to also hide color modes. Why this change?
I know color modes are *probably* not useful without sliders, but they are still 2 separate properties.